### ISSUE TITLE
arrow link does not have a discernible name

### ISSUE THREAD

**Issue by discoform** [discoform/point-zero#6]
![Screen Shot 2021-07-19 at 11 33 24 AM](https://user-images.githubusercontent.com/23062181/126209762-9ef23fb3-dd9b-4511-9569-edbd95144fff.png)

The lack of a discernible name on links is an accessibility issue. 

adding an aria label should address this:

`aria-label="read the next random post"`

**Comment by discoform** [comment]
👍 

![Screen Shot 2021-07-19 at 11 56 51 AM](https://user-images.githubusercontent.com/23062181/126212503-488675ea-88be-4b35-ae36-45dd4b4a6095.png)
